### PR TITLE
[bitnami/odoo] Add database filter env var

### PR DESCRIPTION
**Pull request by kavichu**
### Description of the change

The current [odoo.conf.tpl](https://github.com/bitnami/containers/blob/97a8d24dca0a93d1c5f23cb4fe332fbeb3615c76/bitnami/odoo/16/debian-11/rootfs/opt/bitnami/scripts/odoo/bitnami-templates/odoo.conf.tpl#L15C14-L15C14) template has by a default a wide range of values for database name.

```
dbfilter = .*
```

By adding the env var `ODOO_DATABASE_FILTER` it is possible to filter a specific regex for the database name, this was proposed in [this PR ](https://github.com/bitnami/containers/pull/50435)

### Benefits

It allows to set a regex that matches a specific database name.

### Additional information

I tested using this example configuration:

```
helm upgrade --install -f values.yml \
     --set odooEmail="odoo@example.com" \
     --set odooPassword="password" \
     --set odooSkipInstall="true" \
     --set odooDatabaseFilter="^business_database\$" \
     --namespace default \
     odoo-aratiri ./bitnami-charts/bitnami/odoo
```

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by kavichu** [comment]
Hello, @migruiz4.

The [[bitnami/odoo] Add database filter #50435](https://github.com/bitnami/containers/pull/50435) was merged, now there is an env var `ODOO_DATABASE_FILTER ` to set the `dbfilter` value of `odoo.conf.tpl`.

File: bitnami/odoo/Chart.yaml [hunk 1]
```diff
@@ -34,4 +34,4 @@ maintainers:
 name: odoo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/odoo
-version: 24.0.0
```
Comment (migruiz4) [inline bitnami/odoo/Chart.yaml:37]: Could you please bump the chart version again?

version 24.0.1 has already been released: https://github.com/bitnami/charts/blob/main/bitnami/odoo/Chart.yaml

**Comment by migruiz4** [comment]
Hi @kavichu,

Thank you very much for your contribution! Your changes look good to me, but could you please update the chart version again?

File: bitnami/odoo/Chart.yaml (hunk 1 above)
Comment (kavichu) [reply bitnami/odoo/Chart.yaml:37]: Done, thanks for pointing out.

**Comment by kavichu** [comment]
Hi @migruiz4, I updated the version of the chart.

**Comment by kavichu** [comment]
Hi @migruiz4, I updated the Chart vesion.

File: bitnami/odoo/Chart.yaml [hunk 2]
```diff
@@ -34,4 +34,4 @@ maintainers:
 name: odoo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/odoo
-version: 24.0.4
+version: 24.0.5
```
Comment (migruiz4) [inline bitnami/odoo/Chart.yaml:37]:
```suggestion
version: 24.0.6
```

**Review by migruiz4 — APPROVED** [review]
I'm sorry for the late response @kavichu.

I have bumped the version and resolved conflicts with main branch to reduce the delay.

Thank you for your contribution!

**Pull request merged**